OPHD/States/PlanetSelectState.cpp

```diff
@@ -13,9 +13,13 @@
 #include <NAS2D/Mixer/Mixer.h>
 #include <NAS2D/Renderer/Renderer.h>
 
+#include <cstddef>
+#include <limits>
+
 using namespace NAS2D;
 
-Planet::PlanetType PLANET_TYPE_SELECTION = Planet::PlanetType::None;
+std::size_t planetSelection;
+constexpr std::size_t planetSelectionInvalid = std::numeric_limits<int>::max();
```

Comment:
This is mixing `std::size_t` and `int`.

Actually, if you use `auto` for the variable type here, the return type of `max()` should match the `numeric_limits` template type parameter.

Reply:
good catch.